src/Tx_main.cpp

```diff
@@ -150,6 +153,39 @@ void SendVersionResponse()
   msp.sendPacket(&out, &Serial);
 }
 
+void HandleConfigMsg(mspPacket_t *packet)
+{
+  uint8_t key = packet->readByte();
+  uint8_t value = packet->readByte();
+  switch (key)
+  {
+    case MSP_ELRS_BACKPACK_CONFIG_TLM_MODE:
+      switch (value)
+      {
+        case 0: // TELEM_MODE_OFF
```

Comment:
If you've got an enum then why not use it?

Reply:
I'm happy to change to the named enum values instead.
My reasoning was because this is data from the main firmware that we are parsing, and there's nothing stopping someone coming in later on and adding a new enum option, which re-orders the numbering under the hood. This felt safer. Happy to change it though.